index.html

```diff
@@ -392,18 +397,28 @@ <h3>Use Cases and Requirements</h3>
 containing proofs of claims from any <a>Issuer</a>.
           </li>
           <li>
-Verification shouldn't depend on direct interactions between <a>Issuers</a> and <a>Verifiers</a> . 
+<a>Issuers</a> issue <a>verifiable credentials</a>, and <a>holders</a> store
```

Comment:
Looks like a bad merge here... I think this was removed in https://github.com/w3c/vc-data-model/pull/420. I'll fix it up after the merge.